apis/core/v1alpha1/resource_metadata.go

```diff
@@ -30,4 +30,7 @@ type ResourceMetadata struct {
 	// OwnerAccountID is the AWS Account ID of the account that owns the
 	// backend AWS service API resource.
 	OwnerAccountID *AWSAccountID `json:"ownerAccountID"`
+	// AWSRegion is the AWS region in which the resource is expected to be
+	// managed.
+	AWSRegion *AWSRegion
```

Comment:
Let's just call this field `Region`, since it is assumed to be the AWS region...

Reply:
Changed few seconds after the first push :)